guides/common/modules/proc_securing-the-dhcp-api.adoc

```diff
@@ -0,0 +1,31 @@
+[id="Securing_the_dhcpd_API_{context}"]
+= Securing the dhcpd API
+
+By default, the dhcpd API listens to any host.
+You can add a `omapi_key` to provide basic security.
+
+.Procedure
+. Generate a key:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# yum install -y bind
```

Comment:
At least on EL8 you need bind-utils, not bind for `dnssec-keygen`. IIRC it's different on Debian.

Reply:
Fixed.